packages/protocol/contracts/L2/TaikoL2.sol

```diff
@@ -24,158 +24,102 @@ import { TaikoL2Signer } from "./TaikoL2Signer.sol";
 contract TaikoL2 is EssentialContract, TaikoL2Signer, ICrossChainSync {
     using LibMath for uint256;
 
+    uint64 public constant ETHEREUM_GAS_TARGET = 15 * 1e6;
+    uint64 public constant TAIKO_GAS_TARGET = ETHEREUM_GAS_TARGET * 10;
+    uint64 public constant ADJUSTMENT_QUOTIENT = 8;
+
     struct VerifiedBlock {
         bytes32 blockHash;
         bytes32 signalRoot;
     }
 
-    struct EIP1559Config {
-        uint128 xscale;
-        uint128 yscale;
-        uint32 gasIssuedPerSecond;
-        uint64 gasExcessMax;
-    }
-
     // Mapping from L2 block numbers to their block hashes.
     // All L2 block hashes will be saved in this mapping.
     mapping(uint256 blockId => bytes32 blockHash) private _l2Hashes;
     mapping(uint256 blockId => VerifiedBlock) private _l1VerifiedBlocks;
 
     // A hash to check the integrity of public inputs.
     bytes32 public publicInputHash; // slot 3
-    uint64 public parentTimestamp; // slot 4
+    uint128 public gasExcess; // slot 4
     uint64 public latestSyncedL1Height;
-    uint64 public gasExcess;
 
     uint256[146] private __gap;
 
     // Captures all block variables mentioned in
     // https://docs.soliditylang.org/en/v0.8.20/units-and-global-variables.html
-    event Anchored(
```

Comment:
@davidtaikocha David, I removed may data from this event as those data are available in the block header. Let me know if it's OK.